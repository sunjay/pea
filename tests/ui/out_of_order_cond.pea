fn main() {
    let a = 3;

    if a == 0 { println!(1); }
    else if a == 1 { println!(1); }
    else if a == 2 { println!(1); }
    else { println!(b"uh oh!"); }
    else if a == 3 { println!(1); }
}
