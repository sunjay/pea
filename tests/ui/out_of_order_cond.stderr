[tests/ui/out_of_order_cond.pea:8:5-8:8] error: expected a byte string literal, found: `else`

[tests/ui/out_of_order_cond.pea:9:1] error: expected `struct` or `fn`, found: `}`

error: aborting due to 2 previous errors

